shared/dataflow/change-notes/2024-04-09-alert-provenance.md

```diff
@@ -0,0 +1,4 @@
+---
+category: feature
+---
+* The `PathGraph` result of a data flow computation has been augmented with model provenance information for each of the flow steps.
```

Comment:
Is it the case that users who write custom queries and corresponding qltests may have to update their `.expected` output as a result of this PR? If so, this should be mentioned prominently in the change note to avoid surprises and confusion by users.

Reply:
Here's an amendment: https://github.com/github/codeql/pull/16211